src/core/cff_parser.js

```diff
@@ -644,7 +648,9 @@ var CFFParser = (function CFFParserClosure() {
                                        globalSubrIndex);
         }
         if (state.width !== null) {
```

Comment:
I thought it was strange that the widths were broken in TaroUTR50SortedList112.pdf so I looked in acrobat. Font "QEGUTH+KozMinPro-Regular" is a cid font so we should also look in the sub font dict's private dict for the nominal width and default width. In acrobat there are valid values there. There's some code above here that gets the private dict see `fdArray[fdIndex].privateDict`